flower/command.py

```diff
@@ -158,24 +164,18 @@ def is_flower_envvar(name):
         name[len(ENV_VAR_PREFIX):].lower() in default_options
 
 
-def print_banner(app, ssl):
-    if not options.unix_socket:
-        if options.url_prefix:
-            prefix_str = f'/{options.url_prefix}/'
-        else:
-            prefix_str = ''
-
-        logger.info(
-            "Visit me at http%s://%s:%s%s", 's' if ssl else '',
-            options.address or '0.0.0.0', options.port,
-            prefix_str
-        )
+def print_banner(flower_app):
+    if not flower_app.options.unix_socket:
+        url = flower_app.get_url()
+        logger.info("Visit me at %s", url)
```

Comment:
This is the fix. We get the URL from the Flower app. Before we got it from the Celery app before the server started.